landcover.mss

```diff
@@ -227,6 +227,25 @@
     [way_pixels >= 64] { polygon-gamma: 0.3;  }
   }
 
+  [feature = 'place_village'],
+  [feature = 'place_hamlet'] {
+    [zoom >= 10] {
+      polygon-fill: @built-up-lower-lowzoom;
+      polygon-opacity: 0.3;
```

Comment:
Don't use opacity.

Reply:
As already said - the intention in using opacity was to not obfuscate finer detail. This can be removed if preferred.

Reply:
Given my basic concerns with rendering places like built up areas I'm not sure what changes you should make, except you need to find a way without opacity.